forge-gui/res/cardsfolder/upcoming/unctus_grand_metatect.txt

```diff
@@ -0,0 +1,13 @@
+Name:Unctus, Grand Metatect
+ManaCost:1 U U
+Types:Legendary Artifact Creature Phyrexian Vedalken
+PT:2/4
+S:Mode$ Continuous | Affected$ Creature.Other+Blue+YouCtrl | AddTrigger$ LootTrig | Description$ Other blue creatures you control have "Whenever this creature becomes tapped, draw a card, then discard a card."
+SVar:LootTrig:Mode$ Taps | ValidCard$ Card.Self | Execute$ TrigDraw | TriggerDescription$ Whenever CARDNAME becomes tapped, draw a card, then discard a card.
```

Comment:
`TriggerDescription$ Whenever CARDNAME` should be `TriggerDescription$ Whenever this card`